lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -0,0 +1,164 @@
+                   this.countries = options.teamParams.countries;
+
+                   this.eventAggregator = options.eventAggregator;
+                   _.bindAll(this, "cancelTeam", "createTeam");
+                   this.eventAggregator.bind("cancelTeam", this.cancelTeam);
+                   this.eventAggregator.bind("createTeam", this.createTeam);
+
+                   this.teamNameField = new FieldViews.TextFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Team Name (Required) *"),
+                       valueAttribute: 'name',
+                       bindEvents: false,
+                       helpMessage: gettext("The name that will identify your team")
+                   });
+
+                   this.teamDescriptionField = new FieldViews.TextareaFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Team Description (Required) *"),
+                       valueAttribute: 'description',
+                       editable: 'always',
+                       showMessages: false,
+                       bindEvents: false,
+                       descriptionMessage: gettext("A short description of the team to help other students understand the goals or directives the team is pursuing")
+                   });
+
+                   this.optionalDescriptionField = new FieldViews.ReadonlyFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Optional Characteristics"),
+                       valueAttribute: 'optional_description',
+                       helpMessage: gettext("You can help students find your team by specifying your team's characteristics. The more limitations you add, the fewer students may be interested in joining your group, so choose carefully!")
+                   });
+
+                   this.teamLanguageField = new FieldViews.DropdownFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Language"),
+                       valueAttribute: 'language',
+                       required: false,
+                       showMessages: false,
+                       bindEvents: false,
+                       titleIconName: 'fa-comment-o',
+                       options: this.languages,
+                       helpMessage: gettext("The primary language of the team")
+                   });
+
+                   this.teamCountryField = new FieldViews.DropdownFieldView({
+                       model: new TeamModel(),
+                       title: gettext('Country'),
+                       valueAttribute: 'country',
+                       required: false,
+                       showMessages: false,
+                       bindEvents: false,
+                       titleIconName: 'fa-globe',
+                       options: this.countries,
+                       helpMessage: gettext("The primary country of the team")
```

Comment:
"The country that team members primarily identify with."
@explorerleslie Thoughts?